lib/screens/mobile/settings_screen.dart

```diff
@@ -0,0 +1,45 @@
+import "package:flutter/material.dart";
+import "package:theme_provider/theme_provider.dart";
+
+class SettingsScreen extends StatelessWidget {
+  const SettingsScreen({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        leading: IconButton(
+          onPressed: () {
+            Navigator.pop(context);
+          },
+          icon: const Icon(
+            Icons.arrow_back,
+            color: Colors.blueGrey,
+          ),
+        ),
+      ),
+      body: Column(
+        children: [
+          ListTile(
+            onTap: () async {
+              await showDialog(context: context, builder: (_) => ThemeDialog()).timeout(
+                const Duration(seconds: 2),
+                onTimeout: () {
+                  Navigator.pop(context);
+                },
+              );
```

Comment:
Timeout is not required
```suggestion
              await showDialog(context: context, builder: (_) => ThemeDialog()));
```